sdks/python/src/opik/integrations/dspy/callback.py

```diff
@@ -156,6 +163,21 @@ def _end_trace(self, call_id: str) -> None:
             if self._context_storage.get_trace_data() == trace_data:
                 self._context_storage.set_trace_data(None)
 
+    def _update_with_graph(
```

Comment:
`_try_add_module_graph_to_metadata(module)` origins metadata doesn't need to be passed since it's a class instance attribute.

Reply:
Done